statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/api/request/FiltersWrapper.java

```diff
@@ -10,6 +12,8 @@
 @JsonSerialize
 public class FiltersWrapper {
 
+  @JsonProperty(FilterNames.DATASET_ID)
+  private String datasetId;
```

Comment:
But isn't this a filter too? Why is it separated?
I think we could have it as part of the other filters which makes it more generic and also would support all other functionality that the other filters provide right?
(We could restrict that specific filter in code, to not have breakdowns for example, if we wanted it to but I don't think we need to.)

Reply:
After our chat, we agreed on introducing a structurally "non-breakdownable"  filter type. 
I will do the necessary changes